monitoring/messagesigning/README.md

```diff
@@ -0,0 +1,25 @@
+# Message signing reporting/testing set up
+
+There is an environment variable titled `MESSAGE_SIGNING`. It must be set to `"true"` within `run_locally.sh` and `run_locally_scdsc.sh`. This will ensure that outgoing requests are signed and incoming requests, as well as received responses, have their message signing headers validated.
```

Comment:
I don't see the new configuration flag in Config class below. You may want to add it?

Reply:
This is an environment variable as of right now, set similarly to other env vars (`MOCK_USS_DSS_URL`, etc)

Reply:
Note the discussion [here](https://github.com/interuss/dss/pull/899#discussion_r1023171932); it would probably be a good idea to align on design by completing #899 before working on PRs that refer to wider designs not included in the PR (like this one)